uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -782,6 +787,30 @@ define([
       }
 
       /**
+       * Helper functions to get available statuses depending on the
+       * current request status value.
+       *
+       * @return {Array}
+       */
+      function getAvailableStatusesForCurrentStatus () {
+        var currentStatus = this.getStatusFromValue(this.request.status_id);
+
+        return getAvailableStatusesForStatusName.call(this, currentStatus.name);
+      }
+
+      /**
+       * Helper function that returns an array of the statuses available
+       * for a specific status name.
+       *
+       * @return {Array}
+       */
+      function getAvailableStatusesForStatusName (statusName) {
+        return _.map(availableStatusesMatrix[ statusName ], function (status) {
+          return this.requestStatuses[ status ];
```

Comment:
Please remove spaces `[ status ]` -> `[status]` as well as for *statusName* and other places.